packages/core/src/node/logger-backend-module.ts

```diff
@@ -54,11 +54,11 @@ export function bindLogger(bind: interfaces.Bind, props?: {
  */
 export const loggerBackendModule = new ContainerModule(bind => {
     bind(BackendApplicationContribution).toDynamicValue(ctx =>
-        ({
-            initialize(): void {
-                setRootLogger(ctx.container.get<ILogger>(ILogger));
-            }
-        }));
+    ({
```

Comment:
While we're fixing the formatting, why not do it right?
```
bind(BackendApplicationContribution).toDynamicValue(ctx =>{
    initialize(): void {
        setRootLogger(ctx.container.get<ILogger>(ILogger));
    }
});
```

Reply:
That does not work and gives a. You cannot directly return object without surrounding it with (). Otherwise the compiler will treat it like a function block and throw an error.